groups/groups.yaml

```diff
@@ -183,10 +183,10 @@ groups:
       WhoCanPostMessage: "ANYONE_CAN_POST"
       ReconcileMembers: "true"
     owners:
-      - briangrant@google.com
       - ihor@cncf.io
-      - jorgec@vmware.com
+      - josh@berkus.org
       - killen.bob@gmail.com
```

Comment:
Bob should be removed

Reply:
so just me/ihor/jaice?  OK.  Wasn't clear.

Reply:
Fixed and squash-pushed